api/handlers/org_quota.go

```diff
@@ -0,0 +1,47 @@
+package handlers
+
+import (
+	"net/http"
+	"net/url"
+
+	"code.cloudfoundry.org/korifi/api/presenter"
+	"code.cloudfoundry.org/korifi/api/routing"
+)
+
+const (
+	OrgQuotasPath = "/v3/organization_quotas"
+	OrgQuotaPath  = "/v3/organization_quotas/{guid}"
+)
+
+type CFOrgQuotaRepository interface{}
+
+type OrgQuota struct {
+	apiBaseURL   url.URL
+	orgQuotaRepo CFOrgQuotaRepository
+}
+
+func NewOrgQuota(apiBaseURL url.URL, orgQuotaRepo CFOrgQuotaRepository) *OrgQuota {
+	return &OrgQuota{
+		apiBaseURL:   apiBaseURL,
+		orgQuotaRepo: orgQuotaRepo,
+	}
+}
+
+func (h *OrgQuota) get(r *http.Request) (*routing.Response, error) {
+	return routing.NewResponse(http.StatusOK).WithBody(presenter.ForOrgQuota(h.apiBaseURL)), nil
+}
+
+func (h *OrgQuota) list(r *http.Request) (*routing.Response, error) {
+	return routing.NewResponse(http.StatusOK).WithBody(presenter.ForOrgQuota(h.apiBaseURL)), nil
```

Comment:
any particular reason not to use `presenter.ForList` here? It fills in the pagination "header" for you.